apps/demos/Demos/Menu/Scrolling/Vue/App.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="demo-container">
+    <div class="content">
+      <div class="label">Catalog:</div>
+      <DxMenu
+        :data-source="products"
+        @item-click="itemClick"
+        @submenu-showing="onSubmenuShowing"
+      />
+    </div>
+    <div class="options">
+      <div class="caption">Options</div>
+      <div class="option">
+        <DxCheckBox
+          v-model:value="limitSubmenuHeight"
+          :text="`Limit submenus height by ${SUBMENU_HEIGHT}px`"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+<script setup lang="ts">
+import { ref } from 'vue';
+import DxMenu from 'devextreme-vue/menu';
+import DxCheckBox from 'devextreme-vue/check-box';
+import notify from 'devextreme/ui/notify';
+import service from './data.ts';
+
+const SUBMENU_HEIGHT = 200;
+const products = ref(service.getProducts());
+const limitSubmenuHeight = ref(false);
+
+function itemClick(e: any) {
+  if (!e.itemData.items) {
+    notify(`The "${e.itemData.text}" item was clicked`, 'success', 1500);
+  }
+}
+
+function onSubmenuShowing({ submenuContainer }: any) {
```

Comment:
```suggestion
function onSubmenuShowing({ submenuContainer }: HTMLDivElement) {
```
or HTMLElement, dunno what kind of element is this